security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,47 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-6-0_{context}"]
+== OpenShift Compliance Operator 1.6.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.6.0:
+
+* link:https://access.redhat.com/errata/RHBA-2024:6761[RHBA-2024:6761 - OpenShift Compliance Operator 1.6.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-6-0-new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+* The Compliance Operator now contains supported profiles for Payment Card Industry Data Security Standard (PCI-DSS) version 4. For more information, see xref:../../security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc#compliance-supported-profiles_compliance-operator-supported-profiles[Supported compliance profiles].
+
+* The Compliance Operator now contains supported profiles for Defense Information Systems Agency Security Technical Implementation Guide (DISA STIG) V2R1. For more information, see xref:../../security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc#compliance-supported-profiles_compliance-operator-supported-profiles[Supported compliance profiles].
+
+* A `must-gather` extension is now available for the Compliance Operator installed on `x86` architectures. The `must-gather` tool provides crucial configuration details to Red Hat Customer Support and engineering. For more information, see xref:../../security/compliance_operator/co-support.adoc#compliance-must-gather_co-support[Using the must-gather tool for the Compliance Operator].
```

Comment:
Nit: `x86` architecture
Actually I am not quite sure whether the image works for ppc64le and s390x, as images available there, too.

Reply:
it's cross compiled for all arches. I've used it